### ISSUE TITLE
Detach is not fully working.

### ISSUE THREAD

**Issue by SofiaRedmond** [j-hc/zygisk-detach#37]
**Are you sure you read the [readme](https://github.com/j-hc/zygisk-detach#usage)?**

- Yes

**Android version and skin (MIUI etc... is MIUI still a thing? idk):**

- MiUI 14.0.6, Android 13.

**Screenshot of Play Store page of the app you are trying to detach:**

![Screenshot_2024-03-14-09-58-52-837_com.android.vending.jpg](https://github.com/j-hc/zygisk-detach/assets/86848962/48423c27-54ac-4f9a-a208-2c28895e1474)

**Screenshot of the helper CLI showing detached apps (from Option 2):**

![Screenshot_2024-03-14-14-50-37-076_com.termux.jpg](https://github.com/j-hc/zygisk-detach/assets/86848962/7b0ab8ce-c2d6-4f8f-8254-4623352577cc)

**Logs taken with `su -c logcat -d -s "zygisk-detach" > /sdcard/log.txt`:**

log.txt is empty.

Not providing this information will get your issue closed

The issue is that the app (WhatsApp) appears as a "disabled" app for PlayStore and getting updated even if auto update is disabled, instead as "no installed" app.

On previous version of the module, the app appears (on PlayStore) as is not installed.

**Comment by j-hc** [comment]
on v1.13.0?
and you should take the logs after opening play store.

**Comment by SofiaRedmond** [comment]
> on v1.13.0?
> and you should take the logs after opening play store.

Yes, 1.13.0

Now,  logcat just says:
```
--------- beginning of main
03-15 19:46:19.415 19747 19747 D zygisk-detach: Loaded!
03-15 19:46:20.415 19929 19929 D zygisk-detach: Loaded!
 
```

**Comment by racnes** [comment]
same for me

**Comment by lefuglyduck** [comment]
Same issue, I downgraded to version 1.11.0 and I no longer have this issue.

**Comment by j-hc** [comment]
fixed with v1.14.0

**Comment by aksyanoff** [comment]
Have this issue right now on oneplus 8t, Derpfest 16.2 ROM (Android 16 QPR2)
